tomography_preprocessing/tilt_series_alignment/aretomo/_utils.py

```diff
@@ -33,9 +29,11 @@ def write_relion_tilt_series_alignment_output(
         imod_directory: Path,
         output_star_file: Path,
 ):
-    shifts_px, euler_angles = get_tilt_series_alignment_parameters(imod_directory, tilt_series_id)
-    tilt_image_df[['rlnOriginXAngst', 'rlnOriginYAngst']] = shifts_px * pixel_size
-    tilt_image_df[['rlnAngleRot', 'rlnAngleTilt', 'rlnAnglePsi']] = euler_angles
+    """Write output from a tilt-series alignment experiment."""
+    shifts_px, xf, tlt = get_tilt_series_alignment_parameters(imod_directory, tilt_series_id)
```

Comment:
with the changes suggested above this would become
```suggestion
    xf, tlt = get_tilt_series_alignment_data(imod_directory, tilt_series_id)
    shifts_px = imod_utils.calculate_specimen_shifts(xf)
```